flake.nix

```diff
@@ -304,6 +304,7 @@
         common-cpu-amd-zenpower = import ./common/cpu/amd/zenpower.nix;
         common-cpu-amd-raphael-igpu = import ./common/cpu/amd/raphael/igpu.nix;
         common-cpu-intel = import ./common/cpu/intel;
+        common-cpu-intel-coffee-lake = import ./common/cpu/intel/coffee-lake;
```

Comment:
```suggestion
```

Reply:
We no longer expose new flake attributes for common modules. See: https://github.com/NixOS/nixos-hardware/issues/992

Reply:
You can still import them as a file but there is no stable interface guaranteed. If you want to use them, please consider helping to upstream them.